website/docs/guides/service_principal_client_certificate.html.markdown

```diff
@@ -86,18 +79,50 @@ At this point the newly created Azure Active Directory Application should be ass
 
 ---
 
-### Configuring the Service Principal in Terraform
+## Configuring Terraform to use the Client Certificate
 
-As we've obtained the credentials for this Service Principal - it's possible to configure them in a few different ways.
+Now that we have our Client Certificate uploaded to Azure and ready to use, it's possible to configure Terraform in a few different ways.
 
-When storing the credentials as Environment Variables, for example:
+The provider can be configured to read the certificate bundle from the .pfx file in your filesystem, or alternatively you can pass a base64-encoded copy of the certificate bundle directly to the provider.
+
+### Environment Variables
+
+Our recommended approach is storing the credentials as Environment Variables, for example:
+
+*Reading the certificate bundle from the filesystem*
```

Comment:
That was the intention actually, since this is more of an annotation than a title (which would warrant a header tag rather than bold text). If you don't think that works, we can make these subheaders instead? We have been doing this in AzureAD docs for awhile, see e.g. https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/guides/service_principal_client_certificate#environment-variables or https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/resources/app_role_assignment#example-usage